packages/assert/package.json

```diff
@@ -49,6 +49,9 @@
     "env": {
       "es6": true
     },
+    "globals": {
```

Comment:
So this is necessary to stop the `harden` errors?

Reply:
Yes, to prevent `harden` from triggering eslint.